python/shark_turbine/dynamo/importer.py

```diff
@@ -398,6 +451,8 @@ def import_nodes(self, nodes: Sequence[torch_fx.Node]):
                     elif isinstance(target, TorchOpOverload):
                         # Dispatch to an ATen op.
                         self._import_torch_op_overload(loc, node, target)
+                    elif target in SYMBOLIC_TORCH_OPS or (is_symbolic(node.meta.get('val')) and is_builtin_function_or_method(target)):
```

Comment:
Nit: Use double quotes consistently (with rest of file) unless if a different quote style is called for.